aggregators/merger.go

```diff
@@ -128,7 +117,7 @@ func (m *combinedMetricsMerger) merge(from *aggregationpb.CombinedMetrics) {
 func mergeServiceInstanceGroups(
 	to *serviceMetrics,
 	from []*aggregationpb.KeyedServiceInstanceMetrics,
-	totalTransactionGroupsConstraint, totalServiceTransactionGroupsConstraint, totalSpanGroupsConstraint *constraint,
+	totalTransactionGroupsConstraint, totalServiceTransactionGroupsConstraint, totalSpanGroupsConstraint *constraint.Constraint,
```

Comment:
```suggestion
	constraints constraints,
```
Maybe pass the full constraints here?